src/jobs/enrich-mustache-from-path-filter.yml

```diff
@@ -0,0 +1,53 @@
+  # meant to be a drop in replacement for path-filtering/filter
+  # BUT: meant to be used in a situation where EITHER you
+  # want an easy way to use the rest of this orb's enhance-mustache
+  # functionality OR find that Circle's official path-filtering orb
+  # doesn't work for your use case anymore.
+  #
+  # For exmaple, Circle's path-filtering works great if you want to turn on or off
+  # currently defined workflows or part of a pipeline. But if you have
+  # a situation where you want to create many instances of the SAME
+  # jobs, this is much harder to keep DRY.
+  #
+  # In enrich-mustache-from-path-filter instead of passing parameters
+  # via CircleCI continutation pipeline, it renders the specified Circle
+  # pipeline as a mustache document. This Circle job was created to make the
+  # path from path-filter
+  #
+  # Rules for parameters provided in the mappings section:
+  #   1. these must be JSON literals
+  #   2. If the JSON literal is an array [], if multiple matches are
+  #      found then all the matched values will be concatinated in an array
+  #   3. If a value is not marked as an array then subsequent sets will override
+
+docker:
+  - image: 'cimg/python:3.11'
+parameters:
+  base-revision:
+    type: string
+  config-path:
```

Comment:
can this _also_ be a folder and we apply it to every item? (also useful as a parameter to enrich-mustache command)

Reply:
in most cases, I feel like you'd only want to use this on one file right?

Reply:
_theoretically_ yes, but we're leaning reasonably hard into [Circle's fyaml](https://github.com/CircleCI-Public/fyaml) syntax....... so practically we could use this a few interesting places internally if we had that support